ios/FluentUI/List/ListItem.swift

```diff
@@ -189,6 +189,7 @@ public struct ListItem<LeadingContent: View,
                 accessoryView
             }
             .frame(minHeight: layoutType.minHeight)
+            .opacity(isEnabled ? ListItemTokenSet.enabledAlpha : ListItemTokenSet.disabledAlpha)
```

Comment:
Generally we prefer using a specific color rather than changing the opacity of a control. Changing the opacity can lead to unintentional accessibility issues depending on the background color.

Reply:
hmm, I agree. But I guess how do we go about doing that? The token set only has an alpha specified and doesnt have disabled colors. 

I believe that would be needed to support doing colors, instead of alpha like what is being done for MSFTableViewCell.

Reply:
I guess this is how the TVC handles it, though.